README.md

```diff
@@ -2,10 +2,21 @@
 
 A module for Terasology.
 
-## Planed content
+## Getting Started
 
-A character called gooey will appear. He offers you existing and dangerous quests. However be warned: He might modify your world not in a way you like it!
+- You have to start the game with GooeysQuests activated and in a Perlin world.
+- Walk around until you find a character named Gooey that will ask you something.
+- Depending on your answer, Gooey will change the world adding some structures.
+- Right now there are two available structures, the dungeon and the dwarf hall, but they have different combinations, so every time gooey add the structure, you will see something new.
+- But be careful, maybe you don’t like these changes.
+
+## Adding more structures
+
+If you want to add a new structure, first you have to create the structures templates that will form the structure. Then you will have to create two java classes: a component and a system, typically in the folder `/src/main/java/org/terasology/gooeysQuests/quests`
+##### Naming conventions:
+- Add the name of your structure at the beginning of the name of each structure template that you create
+- The format for java class names is: <YourStructure>QuestComponent and <YourStructure>QuestSystem
```

Comment:
Add the grave/backtick symbol around `<YourStructure>QuestComponent` and `<YourStructure>QuestSystem`.

Reply:
for reference that is this `` ` ``